src/app/component/CheckoutShippingStep/CheckoutShippingStep.component.js

```diff
@@ -158,7 +160,7 @@ class CheckoutShippingStep extends Component {
               type={ type }
               label={ label }
               note={ note }
-              value={ overrideStateValue || stateValue }
+              value={ overrideStateValue || stateValue || (isSelect && DEFAULT_COUNTRY) }
```

Comment:
You can set override value or correct state value initially in this.state declaration

Reply:
Resolved.